pxr/base/lib/work/filter.h

```diff
@@ -107,7 +107,7 @@ WorkParallelFilterN(size_t N,
          &accumVector,
          &starts](size_t begin, size_t end)
            {
-                for(int j = begin; j < end; ++j){
+                for(size_t j = begin; j < end; ++j){
```

Comment:
Thanks for fixing this, these are unused in pxr, so we'll be moving them out soonish.